extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -223,6 +246,15 @@ public boolean isAnyPropertySet() {
     public static class HibernateOrmConfigCacheExpiration {
         /**
          * The maximum time before an object is considered expired.
+         * <p>
+         * To set the maximum idle time, provide the duration (see note on duration’s format below) via the
+         * {@code quarkus.hibernate-orm.cache."<region_name>".expiration.max-idle} (quotes mandatory).
+         * <p>
+         * The format for durations uses the standard {@code java.time.Duration} format.
+         * You can learn more about it in the <a target="_top" href="https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#parse-java.lang.CharSequence-">Duration#parse()</a> javadoc.
```

Comment:
I think this won't be necessary as [duration note](https://github.com/quarkusio/quarkus/blob/master/docs/src/main/asciidoc/duration-format-note.adoc) will be appended at the end of each generated extension doc.

Reply:
Right I did not know about that shared note before. If that is the case though, it might make sense to keep the Java type so people know what this duration note applies to

Reply:
Right on :-)